src/hooks/useDarkMagic.ts

```diff
@@ -28,3 +29,17 @@ const npx = async (arg0: string | undefined) => {
 
   return undefined
 }
+
+const pipx = async (arg0: string | undefined) => {
+  const res = await useFetch(`https://pypi.org/pypi/${arg0}/json`)
```

Comment:
is this how pipx does it?

Reply:
ChatGPT says so.

Reply:
Actually ChatGPT said XML-RPC, but if I wanted JSON that's the variant.

Reply:
I'll cross fingers chat GPT wasn't hallucinating.